store/db/metric/metrics.go

```diff
@@ -96,7 +95,7 @@ func ProvideMetrics() fx.Option {
 
 type Measures struct {
 	fx.In
-	Queries                  metrics.Counter   `name:"db_queries_total"`
-	QueryDurationSeconds     metrics.Histogram `name:"db_query_duration_seconds"`
-	DynamodbConsumedCapacity metrics.Counter   `name:"dynamodb_consumed_capacity_total"`
+	Queries                  *prometheus.CounterVec `name:"db_queries_total"`
+	QueryDurationSeconds     prometheus.ObserverVec `name:"db_query_duration_seconds"`
```

Comment:
this doesn't need to be a pointer like the other vectors?

Reply:
`ObserverVec` is an interface, and `CounterVec` is a struct.